modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -78,6 +78,7 @@
 
 
         $target_path = $base_path  . $fileBase;
+        error_log("Target path is: $target_path");
```

Comment:
Left over debug statement.. this shouldn't be in the error log.

Reply:
Done!